### PR TITLE
updated query to get correct number of running pods

### PR DESCRIPTION
**Pull request by tommyk**
## Description

fix a query that doesn't take `Completed` pods into account

## 🧪  Testing

tested against our local prometheus to verify counts were correct

### CODE REVIEW

File: charts/prometheus/templates/prometheus-alerts-configmap.yaml [hunk 1]
```diff
@@ -1029,7 +1029,7 @@ data:
             runbook_url:
             summary: Too many pods on '{{`{{ $labels.node }}`}}'
           expr: |
-            max(max(kubelet_running_pod_count{job="kubernetes-nodes"}) by(instance) * on(instance) group_left(node) kubelet_node_name{job="kubernetes-nodes"}) by(node) / max(kube_node_status_capacity_pods{job="{{.Release.Name}}-kube-state"}) by(node) > 0.95
+            sum by (node) ( kube_pod_info and on (pod) kube_pod_status_phase{phase="Running"} == 1 ) / on (node) kube_node_status_capacity_pods{job="{{.Release.Name}}-kube-state"} > 0.95
```
Comment (danielhoherd) [inline charts/prometheus/templates/prometheus-alerts-configmap.yaml:1032]: Should this only include running? IE: what about pods that are initializing, crashlooping, etc.?

Comment (danielhoherd) [reply charts/prometheus/templates/prometheus-alerts-configmap.yaml:1032]: Looks like [there are only 5 phases](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase):
- Pending
- Running
- Succeeded
- Failed
- Unknown

I guess "Completed" is "failed" or "succeeded". Should we include "Pending" in this list?

Comment (tommyk) [reply charts/prometheus/templates/prometheus-alerts-configmap.yaml:1032]: after i did this, i found the following about fixing that query on the internets, which they basically have the same thing.  so not really sure about "not running" pods being in the count, thoughts?

https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/443/files

Comment (tommyk) [reply charts/prometheus/templates/prometheus-alerts-configmap.yaml:1032]: one could ask even why 95% of pod capacity is "too many"

Comment (tommyk) [reply charts/prometheus/templates/prometheus-alerts-configmap.yaml:1032]: this also correlates (from my testing) to be the same value as `k describe node <your node> | grep "Non-terminated Pods"`

**Review by danielhoherd — APPROVED** [review]

**Comment by jpweber** [comment]
The original source of this alert is from the project you linked to @tommyk where they made a similar update. Part of me wants to use their change so we continue to match.

**Comment by tommyk** [comment]
> The original source of this alert is from the project you linked to @tommyk where they made a similar update. Part of me wants to use their change so we continue to match.

that is fine, i found it afterwards or i probably would have just used it. i will test and update it in a second.

**Review by jpweber — APPROVED** [review]

**Comment by tommyk** [comment]
> > The original source of this alert is from the project you linked to @tommyk where they made a similar update. Part of me wants to use their change so we continue to match.
> 
> that is fine, i found it afterwards or i probably would have just used it. i will test and update it in a second.

@jpweber updated the query to what comes out of 'kubernetes-mixin'

**Pull request merged**